cvmfs/file_processing/chunk.h

```diff
@@ -93,7 +102,7 @@ class Chunk {
 
   shash::ContextPtr& content_hash_context() { return content_hash_context_; }
   const shash::Any&  content_hash() const { return content_hash_; }
-  z_stream&          zlib_context() { return zlib_context_; }
+  zlib::Compressor*   get_compressor() { return compressor_; }
```

Comment:
Please rename to `compressor()` to match the [coding style guidelines](https://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Function_Names).
Potentially: `UniquePtr<>::weak_ref()` to return a bare pointer here.